skate-plugin/build.gradle.kts

```
@@ -89,7 +89,10 @@ tasks
 // endregion
 
 dependencies {
+  implementation(libs.okhttp)
+  implementation(libs.okhttp.loggingInterceptor)
   implementation(libs.bugsnag) { exclude(group = "org.slf4j") }
+  implementation(project(mapOf("path" to ":tracing")))
```

You can use project accessors for this instead

```suggestion
  implementation(projects.tracing)
```